perfkitbenchmarker/pkb.py

```diff
@@ -206,7 +206,7 @@ def DoRunPhase(benchmark, name, spec, collector, timer):
   logging.info('Running benchmark %s', name)
   for before_run_hook in BEFORE_RUN_HOOKS:
     before_run_hook(benchmark=benchmark, benchmark_spec=spec)
-  with timer.Measure('Benchmark Run'):
+  with vm_util.RunDStatIfConfigured(spec.vms), timer.Measure('Benchmark Run'):
```

Comment:
This will only work for LINUX and not Windows.  For Windows we should look into xperf.